app/components/AmountInput/AmountInput.js

```diff
@@ -1,46 +1,56 @@
-import React, { Component } from 'react'
-import PropTypes from 'prop-types'
+// @flow
+
+import React from 'react'
 import { observer } from 'mobx-react'
 import Flexbox from 'flexbox-react'
 import cx from 'classnames'
 import FontAwesomeIcon from '@fortawesome/react-fontawesome'
 
-import { minimumDecimalPoints, numberWithCommas } from '../../../utils/helpers'
-import { getActiveElement } from '../../../utils/domUtils'
+import { minimumDecimalPoints, numberWithCommas } from '../../utils/helpers'
+import { ref } from '../../utils/domUtils'
 
 import { formatNextAmountDisplay } from './AmountInputUtils'
 
+type Props = {
+  maxDecimal?: number,
```

Comment:
these are currently encoded as optional properties `maxDecimal?:number` maxDecimal can be undefined or a number as opposed to nullable properties `maxDecimal: ?number` maxDecimal can be null or a number. You should be able to remove the `?` from most of the props and assign them with `defaultProps`. This would probably remove the need for the flowfixme in the file ;).  If non of this makes sense - i'll do a PR to address.